pkgs/tools/misc/qmake2cmake/default.nix

```diff
@@ -9,14 +9,15 @@
 , pytestCheckHook
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "qmake2cmake";
-  version = "1.0.5";
+  version = "1.0.6";
 
   src = fetchgit {
     url = "https://codereview.qt-project.org/qt/qmake2cmake";
-    rev = "v${version}";
-    hash = "sha256-6a1CIzHj9kmNgWN6QPNNUbiugkyfSrrIb7Fbz0ocr6o=";
+    # untagged release
+    rev = "bc52a664a5ac794000ce2bd61714cc69f7f54720";
```

Comment:
Let's ping upstream to make a tag.

Reply:
Thank you!

Reply:
I didn't get a reply from upstream, but I see they pushed the tag now and you've already updated the PR to use the tag.